Docs update: React Native (#8627)
PostHog/posthog.com: contents/docs/libraries/js/index.mdx

The call above ensures that every event sent by the user will include `"icecream pref": "vanilla"` and `"team_id": 22`. This way, if you filtered events by property using `icecream_pref = vanilla`, it would display all events captured on that user after the `posthog.register` call, since they all include the specified Super Property.

> Note that this does **not** set the user's properties. This only sets the properties for their events. To store user properties, see the [setting user properties docs](https://posthog.com/docs/product-analytics/user-properties).

Reviewer: ```suggestion
This does **not** set the user's properties. It only sets the properties for their events. To store user properties, see the [setting user properties docs](/docs/product-analytics/user-properties).
```
